manuscript/book.md

```diff
@@ -2582,7 +2582,7 @@ const map2 = map1.set('drink', 'vodka');
 // -> Map({ food: 'pizza', drink: 'vodka' })
 ```
 
-I’m not a big fan of it because it has completely custom API that one has to learn. Also, converting arrays and objects from plain JavaScript to Immutable.js and back every time we need to work with any native JavaScript API or almost any third-party API, is annoying and feels like the Immutable.js creates more problems than it solves.
+I’m not a big fan of it because it has completely custom API that one has to learn. Also, converting arrays and objects from plain JavaScript to Immutable.js and back every time we need to work with any native JavaScript API or almost any third-party API, is annoying and feels like Immutable.js creates more problems than it solves.
 
```

Comment:
Would it be worth mentioning [seamless-immutable](https://github.com/rtfeldman/seamless-immutable) here?

Reply:
I don't think it adds anything here: I think Immer is generally a better choice and I mention Immutable only because it's so popular.